calculator.html

```diff
@@ -0,0 +1,2178 @@
+<!DOCTYPE html>
+<html lang="en-US">
+
+<head>
+  <!-- GA4 Tag -->
+  <!--<script async src="https://www.googletagmanager.com/gtag/js?id=G-LQY2YVNT21"></script><script>window.dataLayer=window.dataLayer || []; function gtag(){dataLayer.push(arguments);}gtag('js', new Date()); gtag('config', 'G-LQY2YVNT21');</script>-->
```

Comment:
Commented this out because of local errors but probably need to uncomment

Reply:
Yeah just the Google Analytics tags to track views and such. Shouldn't get in the way of the calculator, they're identical on all the pages.